Singularity/Singularity/Platforms/PlatformBlank.cs

```diff
@@ -1472,5 +1475,10 @@ public bool MouseButtonReleased(EMouseAction mouseAction, bool withinBounds)
         {
             return true;
         }
+
+        public bool GetBluePrintStatus()
+        {
+            return mIsBlueprint;
```

Comment:
mIsBlueprint either got renamed or never existed, but this is the reason it does not build